### DISCUSSION TITLE
Not ready. Please check your connection problem. Again :')

### DISCUSSION THREAD

**Discussion by neckturtle** [rustdesk/rustdesk#10080] [Q&A]
So I use Win11 and I still can't connect to the public server. I have this problem with only one device. I've added rs-ny.rustdesk.com  to hosts and refreshed/changed DNS, as suggested in other posts. It's also not blocked by my firewall and antivirus. Also I do not know how to self host, as I'm a rookie to this, so I would prefer a solution with the public sever. Obviously, I'm doing something wrong but I can't understand what. Is there something I'm missing? Thanks
![image_2024-11-28_093527291](https://github.com/user-attachments/assets/15f6ed26-5371-45f6-aeb6-8086a2317708)
![image_2024-11-28_093540563](https://github.com/user-attachments/assets/a86dc425-d05d-49b6-b369-24ada5b945dc)

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/1cr8kfv/should_you_selfhost_a_rustdesk_server/
